arm/Microsoft.Web/staticSites/.parameters/github.parameters.json

```diff
@@ -0,0 +1,76 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "name": {
+            "value": "<<namePrefix>>-az-wss-gh-001"
+        },
+        "repositoryUrl": {
+            "value": "https://github.com/MariusStorhaug/static-website"
```

Comment:
Can this be any website? If so I'd suggest to not use yours in case it may every be deleted

Reply:
The solution interacts with the repo+ci/cd solution to create and register a pipeline file stored in the repo. So if we want to keep this test case we need a repo that the solution can connect to with a PAT to a specific repoURI. We can leave this out, but then we only test "Other" with an empty use case, not really testing the full capacity of the module.

I agree however that this is not tied to my github account, so ideally something impersonal. Lets discuss alternatives on this.

Reply:
Removed this scenario as there is no clear way to scale + need to explore more on how it is used.